packages/core/src/App/Components/Elements/VerticalTabs/vertical-tab-headers.jsx

```diff
@@ -20,6 +42,12 @@ class VerticalTabHeaders extends React.PureComponent {
                         key={idx}
                     />
                 ))}
+                <span
+                    style={{
+                        transform: `translate3d(0, ${this.state.top}px, 0)`,
```

Comment:
```suggestion
                        transform: `translate3d(0, ${this.offset_top}px, 0)`,
```

Reply:
@reza-fs `state` is required here because the getter couldn't get the offset value of the element on mount (it returns undefined), so it needs to be in re-calculated in the next lifecycle (componentDidUpdate)